Gemfile

```diff
@@ -278,6 +278,7 @@ group :test do
   gem "capybara_accessible_selectors", git: "https://github.com/citizensadvice/capybara_accessible_selectors", tag: "v0.12.0"
   gem "capybara-screenshot", "~> 1.0.17"
   gem "cuprite", "~> 0.15.0"
+  gem "ferrum", github: "toy/ferrum", ref: "mouse-events-buttons-property-0.15"
```

Comment:
Great job fixing the bug within ferrum. As soon as a new version with [your merged PR](https://github.com/rubycdp/ferrum/pull/509) is released, we can use the original gem once again. 🚀 Do you happen to know when they plan to release a new version?

Reply:
No idea, I only saw that allowing ferrum dependency to be 0.16.* was merged in cuprite.

Not required for this PR, but I created rubycdp/cuprite#288 to scroll to destination when dragging.